Shark.swift

```diff
@@ -4,7 +4,10 @@ import Foundation
 
 //A simple counted set implementation that uses a dictionary for storage.
 struct CountedSet<Element: Hashable>: Sequence {
-    private var backingDictionary = [Element : Int]()
+    
```

Comment:
Nit: Can you get rid of the empty line here and move the `typealias` declaration to the top line?

Reply:
OK, I  have fixed it.